spatialos/UnrealGDK
# UNR-1402: Crash when attempting to replicate a non supported type

**Contributions**: We are not currently taking public contributions - see our [contributions](CONTRIBUTING.md) policy. However, we are accepting issues and we do want your [feedback](../README.md#give-us-feedback).

-------

#### Description
Unreal produces a warning (rather than crashing the game) if the game attempts to replicate an unsupported type (such as TMap). This PR changes the GDK to match this behaviour.

#### Release note
Attempting to replicate unsupported types (such as TMap) produce a log error rather than crashing the game.

#### Tests
Tested by replicating a TMap in GDKCharacter in the example project and verifying that the game no longer crashed but produced an error.

How can this be verified by QA?

Create an actor that contains a replicated TMap property and play with spatial. The game should run normally with an error message in the output.

#### Primary reviewers
@joshuahuburn @improbable-valentyn
